compatibility-kit/javascript/features/data-tables/data-tables.ndjson

```diff
@@ -3,12 +3,12 @@
 {"pickle":{"id":"11","uri":"features/data-tables/data-tables.feature","name":"transposed table","language":"en","steps":[{"text":"the following table is transposed:","argument":{"dataTable":{"rows":[{"cells":[{"value":"a"},{"value":"b"}]},{"cells":[{"value":"1"},{"value":"2"}]}]}},"id":"9","astNodeIds":["4"]},{"text":"it should be:","argument":{"dataTable":{"rows":[{"cells":[{"value":"a"},{"value":"1"}]},{"cells":[{"value":"b"},{"value":"2"}]}]}},"id":"10","astNodeIds":["7"]}],"astNodeIds":["8"]}}
 {"stepDefinition":{"id":"0","pattern":{"source":"the following table is transposed:","type":"CUCUMBER_EXPRESSION"},"sourceReference":{"uri":"features/data-tables/data-tables.ts","location":{"line":5}}}}
 {"stepDefinition":{"id":"1","pattern":{"source":"it should be:","type":"CUCUMBER_EXPRESSION"},"sourceReference":{"uri":"features/data-tables/data-tables.ts","location":{"line":9}}}}
-{"testRunStarted":{"timestamp":{}}}
+{"testRunStarted":{"timestamp":{"seconds":"0","nanos":0}}}
```

Comment:
This is odd. @vincent-psarga why was it empty before and why is it not anymore?

Reply:
good question. I don't recall regenerating the golden JSONs with the tweak we had made on `protobuf.js` but it may have happened.